Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM wsbu/toolchain-native:v0.3.8
+FROM wsbu/toolchain-native:v0.3.10
```

Comment:
After creating 0.3.9, I discovered another Conan bug and opted not to bother updating toolchain-linaro. That is why this version jumps two instead of one.